File: code_samples/node/v1/setColors.js
const { response } = await sdk.setAppearances({
  surfaceAccent: {
    colorDarkMode: "#ff0000", //🟥
    colorLightMode: "#ff0000",
  },
  tintAccent: {
    colorDarkMode: "#00ff00 ", //🟩
    colorLightMode: "#00ff00",
  },
  tintOnAccent: {
    colorDarkMode: "#0000ff", //🟦
    colorLightMode: "#0000ff",
  },
});
if (response.status === 201) {
  console.log("Logo was successfully saved.");
}

File: code_samples/python/v1/set_colors.py
from tatrapayplus.client import TBPlusSDK
from tatrapayplus.models import *

client = TBPlusSDK(
    "your-client-id",
    "your-client-secret"
)

appearance_data = AppearanceRequest(
    theme=AppearanceRequestTheme.SYSTEM,
    surface_accent=ColorAttribute(color_dark_mode="#fff", color_light_mode="#fff"),
    tint_accent=ColorAttribute(color_dark_mode="#fff", color_light_mode="#fff"),
    tint_on_accent=ColorAttribute(color_dark_mode="#fff", color_light_mode="#fff"),
)
response = client.set_appearance(appearance_data)
